examples/angular/src/pages/ui/components/authenticator/sign-up-with-phone/sign-up-with-phone.component.ts

```diff
@@ -2,19 +2,20 @@ import { Component, NgZone, OnInit } from '@angular/core';
 
 import { Amplify } from 'aws-amplify';
 import awsExports from '@environments/auth-with-phone-number/src/aws-exports';
-Amplify.configure({
-  ...awsExports,
-  auth: {
-    login_mechanisms: ['phone_number'],
-  },
-});
 
 @Component({
   selector: 'sign-up-with-phone',
   templateUrl: 'sign-up-with-phone.component.html',
 })
 export class SignUpWithPhoneComponent implements OnInit {
-  constructor() {}
+  constructor() {
+    Amplify.configure({
```

Comment:
Typo: this configure needs to be inside constructor to ensure `Amplify` is only configured when this component renders.

Reply:
Yeah, I noticed this too, on a couple of examples where I had to move it into the constructor. I'm not sure why

Reply:
My understanding is, because it's a single-page app that's executing all the imported code, `Amplify.configure` would be called as a side-effect of each component, creating tons of conflicts.

Instead, having it within `constructor` at least makes this "refresh-friendly" so that we configure it _once_ per component (assuming 1 component is rendered at a time)

Reply:
This is interesting. @ErikCH you noticed this strictly for the Angular examples? I almost kind of like this because I could imagine a way to have a common frontend implementation where we dynamically pass in aws-exports for different environments as needed by e2e tests. Just spitballing, I don't know if that would actually work. It would clean up our examples a lot though since they're all mostly just a simple `Authenticator` component.